kornia/augmentation/container/augment.py

```diff
@@ -26,28 +26,33 @@ class AugmentationSequential(ImageSequential):
 
```

Comment:
@shijianjian [the image links of this class (L22 and L24)](https://kornia.readthedocs.io/en/latest/augmentation.container.html#kornia.augmentation.container.AugmentationSequential) are broken, should we remove them or it is an easy fix?

Reply:
emm... somebody removed the images in the tutorial repo...

Reply:
Can you add the sequential example images in this repo directly? Code can be found from https://kornia-tutorials.readthedocs.io/en/latest/data_patch_sequential.html.

Reply:
I have added it to the generate images script, just check if these example makes sense:

![AugmentationSequential](https://user-images.githubusercontent.com/20444345/214102085-32a75e66-ac67-459f-9c0d-f8a0976e9247.png)

![PatchSequential](https://user-images.githubusercontent.com/20444345/214105595-5d7f2235-5ba4-4ebb-9dd7-98d75bfc651b.png)

Reply:
Great. I think the grid `4x4` can look better for `PatchSequential`.